frontend/src/pages/home/HomePage.tsx

```diff
@@ -1,30 +1,82 @@
-import reactLogo from '../../assets/react.svg'
-import viteLogo from '/vite.svg'
-import { useState } from "react"
-
 export default function HomePage() {
-    const [count, setCount] = useState(0)
-
-    return (<>
-        <div>
-            <a href="https://vitejs.dev" target="_blank">
-                <img src={viteLogo} className="logo" alt="Vite logo" />
+  return (
+    <div className="bg-white">
+      <header className="absolute inset-x-0 top-0 z-50">
+        <nav
+          aria-label="Global"
+          className="flex items-center justify-between p-6 lg:px-8"
+        >
+          <div className="flex lg:flex-1">
+            <a href="/" className="-m-1.5 p-1.5">
+              <p className="font-black italic text-2xl"> RateMyPeer</p>
```

Comment:
Hey Mouhamed, great job so far! Can you rename all instances of RateMyPeer to RateMyPeers as I think that name is a better website name? Thanks!